src/main/kotlin/com/team4099/robot2020/config/ControlBoard.kt

```diff
@@ -25,4 +25,10 @@ object ControlBoard {
 
     val wristHorizontal: Boolean
         get() = operator.rightShoulderButton
+
+    val climberUp: Boolean
```

Comment:
map this to a button on each of the controllers so that we don't pull a 2363 and accidentally activate climber